deploy/00_resolve_keep_token_staking.ts

```diff
@@ -0,0 +1,31 @@
+import { HardhatRuntimeEnvironment, HardhatNetworkConfig } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments, helpers } = hre
+  const { log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const KeepTokenStaking = await deployments.getOrNull("KeepTokenStaking")
```

Comment:
Will that work if staking contract in `keep-core` is called `TokenStaking`? Won't we aleways get null for `ropsten` and `mainnet` envs?

Reply:
For mainnet we stored a dependency as `external/mainnet/KeepTokenStaking.json`. For ropsten we need to figure out the best way to handle this, as the name `TokenStaking` may clash between `keep-core` and `threshold-network` packages. I propose we leave handling ropsten for another PR, as there are more considerations for integrating testnet deployments.

Reply:
Could we abstain from merging the PR until we have the ropsten deployment figured out? I would like to avoid deployment failures in the eventuality that we're faced with the need to run the ropsten deployment in the following days.

Reply:
30e1e644830953d5fdd8b073f0494f244ec3c961

Reply:
The stubs are for NuCypher contracts, but we still have a problem with deployment of `KeepTokenStaking`. When trying to run ropsten deployment, I'm getting:
```
➜  solidity-contracts git:(token-staking-deployment) ✗ yarn deploy --network ropsten
yarn run v1.22.11
$ hardhat deploy --export export.json --network ropsten
Nothing to compile
using existing KeepToken at 0x29DDc999E81E585D500BcA2FD2A8bcA677680e72
An unexpected error occurred:

Error: ERROR processing /Users/michalina/workspace2/solidity-contracts/deploy/00_resolve_keep_token_staking.ts:
Error: deployed KeepTokenStaking contract not found
    at Object.<anonymous> (/Users/michalina/workspace2/solidity-contracts/deploy/00_resolve_keep_token_staking.ts:17:11)
```

Reply:
🤦🏻‍♂️ Should be good after 3e9ee4bafe2cf7cc481e94c3ac8021d2446e38e9